container/root/etc/services-available/php-fpm/.command

```diff
@@ -3,13 +3,13 @@
 # Ensures that a failure of php-fpm doesn't return with a 0, even if the right-hand side does
 set -o pipefail
 
-# IMPORTANT: PHP 7.3+ optionally allows undecorated stdout/stderr, removing pipe magic requirement
-UPGRADE_COMMAND=`grep "decorate_workers_output = no" $CONF_FPMPOOL`
+# IMPORTANT: PHP 7.3+ optionally allows undecorated stdout/stderr, removes requirement for legacy post-filtering
+NATIVE_COMMAND=$(grep "decorate_workers_output = no" "$CONF_FPMPOOL")
 
-if [ $UPGRADE_COMMAND ]; then
+if [ -n "$NATIVE_COMMAND" ]; then
```

Comment:
no, i'm not sure i want to use string matching vs. return code. It might be late, but I couldn't find *any* combination of `[` or `[[` and error code checking that actually worked.